test/app-tap/datetime.test.lua

```diff
@@ -1830,6 +1830,565 @@ test:test("Parse strptime format", function(test)
+            fmt = '%r',
+        },
+        {
+            buf = '12:32:01 PM',
+            fmt = '%r',
+        },
+        -- %R, same as "%H:%M".
+        {
+            buf = '18:20',
+            fmt = '%R',
+        },
+        -- %s
+        {
+            buf = '26-08-2024 1724630400',
+            fmt = '%d-%m-%Y %s',
+        },
+        -- %S
+        {
+            buf = '00',
+            fmt = '%S',
+        },
+        {
+            buf = '59',
+            fmt = '%S',
+        },
+        -- %t
+        {
+            buf = '26-08-2024 \t',
+            fmt = '%d-%m-%Y %t',
+        },
+        -- %T, same as "%H:%M:%S".
+        {
+            buf = '00:00:00',
+            fmt = '%T',
+        },
+        {
+            buf = '23:59:59',
+            fmt = '%T',
+        },
+        -- %u
+        -- Works for 4 ('Thursday') only when date is not
+        -- complete.
+        {
+            buf = '4',
+            fmt = '%u',
+        },
+        {
+            buf = '30-08-2024 5',
+            fmt = '%d-%m-%Y %u',
+        },
+        {
+            buf = '26-08-2024 1',
+            fmt = '%d-%m-%Y %u',
+        },
+        -- %U
+        {
+            buf = '01-01-2024 00',
+            fmt = '%d-%m-%Y %U',
+        },
+        {
+            buf = '31-12-2024 52', -- "00 to 53"?
+            fmt = '%d-%m-%Y %U',
+        },
+        -- %v, same as "%e-%b-%".
+        {
+            buf = ' 1-Jan-1970',
+            fmt = '%v',
+        },
+        -- %V, unsupported by strptime(3).
+        -- %w
+        -- Works for 4 (Thursday) only when date is not complete.
+        {
+            buf = '4',
+            fmt = '%w',
+        },
+        {
+            buf = '26-08-2024 1',
+            fmt = '%d-%m-%Y %w',
+        },
+        -- %W
+        {
+            buf = '26-08-2024 35',
+            fmt = '%d-%m-%Y %W',
+        },
+        -- %x
+        {
+            buf = '01/01/70',
+            fmt = '%x',
+        },
+        -- %X
+        {
+            buf = '00:00:00',
+            fmt = '%X',
+        },
+        {
+            buf = '23:59:59',
+            fmt = '%X',
+        },
+        -- %y
+        {
+            buf = '00',
+            fmt = '%y',
+        },
+        {
+            buf = '99',
+            fmt = '%y',
+        },
+        -- %Y
+        {
+            buf = '0000',
+            fmt = '%Y',
+        },
+        {
+            buf = '3000',
+            fmt = '%Y',
+        },
+        -- %z
+        {
+            buf = '+0403',
+            fmt = '%z',
+        },
+        {
+            buf = '-0403',
+            fmt = '%z',
+        },
+        -- %Z
+        {
+            buf = 'MSK',
+            fmt = '%Z',
+        },
+        -- %+
+        {
+            buf = 'Mon Aug 26 00:00:00  2024',
+            fmt = '%+',
+        },
+        -- %%
+        {
+            buf = '26-08-2024 %',
+            fmt = '%d-%m-%Y %%',
+        },
+        -- Mixed.
```

Comment:
Side note: What is the principle for "Mixed" tests? Just some most popular use cases? If there is a link for them, it would be nice to mention it here.

Reply:
> Side note: What is the principle for "Mixed" tests?

there is no idea behind these testcases, I've stolen these testcases from another testsuites for `datetime.parse`.